packages/forms/src/UIForm/fields/Text/PasswordWidget/index.js

```diff
@@ -0,0 +1,4 @@
+import Password from '@talend/design-system/lib/components/Form/Field/Input/Input.Password';
```

Comment:
this embed the DS in the bundle of forms
https://unpkg.com/@talend/react-forms@6.40.0/dist/TalendReactForms.js.report.html